cabal-install/Distribution/Client/InstallPlan.hs

```diff
@@ -596,9 +596,9 @@ processingInvariant plan (Processing processingSet completedSet failedSet) =
  && all (isJust . flip Graph.lookup (planIndex plan)) (Set.toList failedSet)
  && noIntersection processingSet completedSet
  && noIntersection processingSet failedSet
+ --   intersection processingClosure failedSet is quite possible however
```

Comment:
Well, the comment makes sense in the context of the patch, but it does get replaced by a better comment later. So it does no harm.